README.md

```diff
@@ -64,9 +38,7 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- This website was made by Wander Gonzalez in 2022 based on Cindy Shin template. Educational Purposes only.
```

Comment:
- Kindly add the link to the original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07) in your readme. The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author.